db/schema.rb

```diff
@@ -31,6 +31,18 @@
   add_index "activities", ["recipient_id", "recipient_type"], name: "index_activities_on_recipient_id_and_recipient_type", using: :btree
   add_index "activities", ["trackable_id", "trackable_type"], name: "index_activities_on_trackable_id_and_trackable_type", using: :btree
 
+  create_table "comments", force: :cascade do |t|
+    t.string   "title",         limit: 255
```

Comment:
As I said in my big comment, I don't like comments having a `title`. I'd remove this.